packages/NodeTypeResolver/TypeAnalyzer/ArrayTypeAnalyzer.php

```diff
@@ -97,10 +141,6 @@ private function isPropertyFetchWithArrayDefault(Node $node): bool
         }
```

Comment:
this is not needed as called inside `PropertyFetch` and `StaticPropertyFetch` check.